src/components/Dissolution/CompleteResolution.vue

```diff
@@ -251,45 +393,79 @@
     </section>
 
     <!-- Confirm Resolution -->
-    <section id="confirm-resolution-section" class="mt-10">
+    <section
+      id="confirm-resolution-section"
+      class="mt-10"
+    >
       <header id="resolution-confirm-header">
-        <h2>{{getCreateResolutionResource.confirmSection.header}}</h2>
+        <h2>{{ getCreateResolutionResource.confirmSection.header }}</h2>
       </header>
 
-      <div class="mt-4" :class="{ 'invalid-section': isInvalid }">
-        <v-card flat id="confirm-resolution-card" class="py-8 px-6">
+      <div
+        class="mt-4"
+        :class="{ 'invalid-section': isInvalid }"
+      >
+        <v-card
+          id="confirm-resolution-card"
+          flat
+          class="py-8 px-6"
+        >
           <v-form ref="confirmResolutionChkFormRef">
             <v-checkbox
-              ref="confirmResolutionChkRef"
               id="chk-confirm-resolution"
-              class="chk-resolution mt-0 pt-0"
+              ref="confirmResolutionChkRef"
               v-model="resolutionConfirmed"
+              class="chk-resolution mt-0 pt-0"
               hide-details
               :rules="confirmCompletionResolution"
               @change="onResolutionConfirmedChange($event)"
             >
-              <template v-slot:label>
+              <template #label>
```

Comment:
Can you test this to make sure it's OK? It works fine for Filings UI and Edit UI so I guess it should be fine here also.

Reply:
Yeah, I tested it and it works.

![Screenshot 2023-06-21 120307](https://github.com/bcgov/business-create-ui/assets/60866283/bccf0435-636e-40a4-92ef-3cc0ac7ef8f5)

Also, you can find reference here that the shorthand of `v-slot` is `#`:
https://vuejs.org/guide/components/slots.html#named-slots

Reply:
^^ Careful with the Vue docs -- some are for Vue3 and others are for Vue2.

Reply:
Great, thanks! Resolving.